src/OpenApi/src/Schemas/OpenApiJsonSchema.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.OpenApi.Models;
+
+[JsonConverter(typeof(JsonConverter))]
+internal sealed partial class OpenApiJsonSchema(OpenApiSchema schema)
+{
+    /// <summary>
+    /// Represents the OpenAPI schema that this instance represents.
+    /// </summary>
+    public OpenApiSchema Schema { get; } = schema;
+
+    internal sealed class JsonConverter : JsonConverter<OpenApiJsonSchema>
+    {
+        public override OpenApiJsonSchema? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            var schema = new OpenApiSchema();
+            if (reader.TokenType != JsonTokenType.StartObject)
+            {
+                throw new JsonException("Expected StartObject token to represent beginning of schema.");
+            }
+            reader.Read();
+            do
+            {
+                switch (reader.TokenType)
+                {
+                    case JsonTokenType.PropertyName:
+                        var propertyName = reader.GetString() ?? throw new JsonException("Encountered unexpected missing property name.");
+                        ReadProperty(ref reader, propertyName, schema, options);
+                        break;
+                    case JsonTokenType.EndObject:
+                        return new OpenApiJsonSchema(schema);
+                    default:
+                        continue;
```

Comment:
What's an example of something we'd skip here?  From the enum, it seems like it's most likely to be a comment?

Reply:
Yep, comment is the most likely case here. The object we are reading here is the schema returned by STJ, which I expect to be generally well-formed, so there's a risk of wacky scenarios like a `{false}` to handle.